test/rules-engine/helper.test.js

```diff
@@ -48,7 +62,7 @@ describe('Rules engine helper', () => {
     })
 
     const runPromise = rulesEngineHelper.fullRun(action, getSampleParcel(), actionData)
-    successCallback()
+    successCallback(undefined, almanac)
```

Comment:
I'm confused by this callback handling. Could we not simply call the callback within the mock implementation for `rulesEngine.doFullRun`, rather than assigning the callback to a variable and calling it here?

Reply:
That would perhaps lead to a simpler mock implementation - what I've done here is simply my initial thought process to mimic the behaviour of the rules engine as closely as possible, where the success callback is invoked before the promise is resolved, and I've never refactored since then. Probably best left as a bit of tech debt for now.